### PR TITLE
fix style_repaint oss question,change to use dashscope

### PR DESCRIPTION
**Pull request by wangyijunlyy**

### CODE REVIEW

File: modelscope_agent/tools/localfile2url_utils/localfile2url.py [hunk 1]
```diff
@@ -1,48 +1,26 @@
...
+from urllib.parse import unquote_plus, urlparse
+
+from dashscope.common.error import InvalidInput, UploadFileException
+from dashscope.utils.oss_utils import OssUtils
+
+FILE_PATH_SCHEMA = 'file://'
+
+
+def get_upload_url(model: str, upload_path: str, api_key: str):
+    if upload_path.startswith(FILE_PATH_SCHEMA):
```
Comment (wenmengzhou) [inline modelscope_agent/tools/localfile2url_utils/localfile2url.py]: add doc str for this function

File: modelscope_agent/tools/localfile2url_utils/localfile2url.py [hunk 2]
```diff
@@ -1,48 +1,26 @@
...
-    return file_url
+from urllib.parse import unquote_plus, urlparse
+
+from dashscope.common.error import InvalidInput, UploadFileException
+from dashscope.utils.oss_utils import OssUtils
+
+FILE_PATH_SCHEMA = 'file://'
+
+
+def get_upload_url(model: str, upload_path: str, api_key: str):
```
Comment (wenmengzhou) [inline modelscope_agent/tools/localfile2url_utils/localfile2url.py]: upload_path -> file_to_upload

File: modelscope_agent/tools/style_repaint.py [hunk 3]
```diff
@@ -112,7 +114,12 @@ def _remote_parse_input(self, *args, **kwargs):
         image_path = kwargs['input'].pop('image_path', None)
         if image_path and image_path.endswith(('.jpeg', '.png', '.jpg')):
             # 生成 image_url，然后设置到 kwargs['input'] 中
-            image_url = path2url(image_path, f'{self.name}/{image_path}')
+            # 复用dashscope公共oss
+            image_path = f'file://{os.path.join(WORK_DIR,image_path)}'
+            image_url = get_upload_url(
+                model='qwen-vl-plus',
```
Comment (wenmengzhou) [inline modelscope_agent/tools/style_repaint.py]: add a comment here why we use qwen-vl-plus model

**Comment by wangyijunlyy** [comment]
model 的注释需要审核一下

**Review by wenmengzhou — APPROVED** [review]

**Pull request merged**